OuterWildsRandomSpeedrun/OuterWildsRandomSpeedrun.cs

```diff
@@ -118,6 +120,11 @@ private void Update()
             _timerPrompt.SetText($"<color=#{ColorUtility.ToHtmlStringRGB(OW_ORANGE_COLOR)}>{elapsedStr}</color>");
         }
 
+        private void OnWakeUp()
+        {
+            Locator.GetPlayerBody().SetVelocity(_spawnPoint.GetPointVelocity());
```

Comment:
Could we call GetSpawnPointByName & use _spawnPointName to avoid needing the new class field? (I figure we're only doing this for 1 frame, so should be insignificant perf-wise)

Reply:
we don't have the spawner either, so I figured a field that mirrors another field we have (_goalPointName) was better

Reply:
Ah, didn't notice it depended on the list of spawn points. LGTM